server/utils/httputils/utils_test.go

```diff
@@ -28,15 +28,15 @@ func TestNormalizeRemoteBaseURL(t *testing.T) {
 
 		// Errors
 		{"[jdsh", "", "",
-			`parse //[jdsh: missing ']' in host`},
+			`parse "//[jdsh": missing ']' in host`},
 		{"/mmtest", "", "",
 			`Invalid URL, no hostname: "/mmtest"`},
 		{"/mmtest/", "", "",
 			`Invalid URL, no hostname: "/mmtest/"`},
 		{"http:/mmtest/", "", "",
 			`Invalid URL, no hostname: "http:/mmtest/"`},
 		{"hƒƒp://xyz.com", "", "",
-			`parse hƒƒp://xyz.com: first path segment in URL cannot contain colon`},
+			`parse "hƒƒp://xyz.com": first path segment in URL cannot contain colon`},
```

Comment:
What are these ƒƒ characters? It seems like the ƒƒ is the only reason that this test passes, because otherwise the url is valid.

Reply:
I have no idea :man_shrugging: